libtenzir/builtins/aspects/index.cpp

```diff
@@ -0,0 +1,67 @@
+//    _   _____   __________
+//   | | / / _ | / __/_  __/     Visibility
+//   | |/ / __ |_\ \  / /          Across
+//   |___/_/ |_/___/ /_/       Space and Time
+//
+// SPDX-FileCopyrightText: (c) 2024 The Tenzir Contributors
+// SPDX-License-Identifier: BSD-3-Clause
+
+#include <tenzir/argument_parser.hpp>
+#include <tenzir/node_control.hpp>
+#include <tenzir/plugin.hpp>
+#include <tenzir/series_builder.hpp>
+#include <tenzir/status.hpp>
+
+#include <caf/scoped_actor.hpp>
+
+namespace tenzir::plugins::index {
+
+namespace {
+
+class plugin final : public virtual aspect_plugin {
+public:
+  auto name() const -> std::string override {
+    return "index";
+  }
```

Comment:
Nit: The name `index` is a bit of a lie, in reality this would be `index-aspect` or `show-index` and have a separate `aspect_name`.

Reply:
We do the same for all the other plugins, where we only use a plugin name that's different from the operator/format/connector/aspect/component name when it would otherwise clash. It's easy to change then. Let's just keep this as-is now.